File: app/src/main/java/br/com/popularmovies/services/movieService/response/MovieReviews.java
package br.com.popularmovies.services.movieService.response;

import com.squareup.moshi.Json;

import java.util.List;

public class MovieReviews {
    @Json(name = "results")
    private List<MovieReview> reviews;

    public List<MovieReview> getReviews() {
        return reviews;
    }
}

File: app/src/main/java/br/com/popularmovies/services/movieService/response/MovieReviews.kt
package br.com.popularmovies.services.movieService.response

import android.os.Parcelable
import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import kotlinx.android.parcel.Parcelize

@Parcelize
@JsonClass(generateAdapter = true)
data class MovieReviews(@Json(name = "results") val reviewDtos: List<MovieReviewDto>) : Parcelable
